sdk/inc/azure/storage/az_storage_blobs.h

```diff
@@ -76,17 +84,20 @@ AZ_NODISCARD az_result az_storage_blobs_blob_client_init(
     void* credential,
     az_storage_blobs_blob_client_options* options);
 
+/**
+ * @brief Azure Storage Blobs Blob upload options.
+ * @remark Reserved for future use
+ */
 typedef struct
 {
   struct
   {
-    az_span option;
+    az_span unused;
```

Comment:
nit: Outside of the scope of this PR, but I was told to use `bool` in similar places elsewhere.